.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Ask git to automatically ensure that text files have posix-style line endings.
+* text eol=lf
+
+# Git can usually detect binary files, but lets mark the binary files we have
+# in the repository as binary just to be safe.
+*.arrow binary
+*.db binary
+*.jpg binary
+*.mp4 binary
+*.parquet binary
+*.png binary
+*.tar binary
+*.tgz binary
+*.webp binary
+*.xlsx binary
+*.zip binary
```

Comment:
We don't plan to run the repo under Windows, except for unit tests—and in that case I'd rather see a failure if something changes. Can we just remove the gitattributes file and live with whatever this results in?

Reply:
I didn't add this to support local development; it is required for Windows CI. I could remove the extra binary attributes and just trust Git to handle binary detection.